config/initializers/content_security_policy.rb

```diff
@@ -20,9 +20,9 @@
                     'https://www.googleapis.com', 'https://s3.amazonaws.com', 'https://data.broadinstitute.org', 'https://portals.broadinstitute.org',
                     'https://us.input.tcell.insight.rapid7.com', 'https://api.tcell.io', 'https://us.browser.tcell.insight.rapid7.com',
                     'https://us.agent.tcell.insight.rapid7.com', 'https://us.jsagent.tcell.insight.rapid7.com', 'https://accounts.google.com',
-                    'https://bam.nr-data.net',
-                    'https://terra-bard-dev.appspot.com', 'https://terra-bard-alpha.appspot.com', 'https://terra-bard-prod.appspot.com',
-                    'https://rest.ensembl.org']
+                    'https://bam.nr-data.net', 'https://terra-bard-dev.appspot.com', 'https://terra-bard-alpha.appspot.com',
+                    'https://terra-bard-prod.appspot.com', 'https://rest.ensembl.org', 'https://eutils.ncbi.nlm.nih.gov', 'https://mygene.info',
+                    'https://webservice.wikipathways.org']
```

Comment:
I'm sure these sites are safe, but I think our practice is that changes to the CSP move the ticket to "Moderate risk", and we should be sure to run these third parties by infoSec.

Reply:
Good point.  I just requested security review [here](https://broadworkbench.atlassian.net/browse/SCP-2363?focusedCommentId=35169&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-35169), per the new CSP policy we floated last week.

Reply:
Security review [passed](https://broadworkbench.atlassian.net/browse/SCP-2363?focusedCommentId=35224&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-35224) -- AppSec agrees this is secure.